src/main/webapp/i18n/en/global.json

```diff
@@ -1,6 +1,6 @@
 {
   "global": {
-    "title": "ServiceNet",
+    "title": " ServiceNet",
```

Comment:
Was this intentional? If yes, the space does not belong here

Reply:
It was intentional, to add a space after the logo in the header in the quickest way. Although, I can handle it with CSS.